lib/api_me.rb

```diff
@@ -166,7 +166,11 @@ def filter_scope(scope)
   end
 
   def sort_scope(scope, params)
```

Comment:
remove `params` arg, this parameter isn't used anymore. Let's remove commented out code too :)